Product/pom.xml

```diff
@@ -1132,6 +1133,11 @@
         <version>1.4.1</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
Agree with Minh.  The dependency was in a commons section in the pom, now it's moved to the bottom.  Better organized before.